test/plugins/commands.test.js

```diff
@@ -8,6 +8,7 @@ const urlLib = require('url');
 const hoek = require('hoek');
 const testcommand = require('./data/command.json');
 const testcommands = require('./data/commands.json');
+const testcommandversions = require('./data/commandVersions.json');
```

Comment:
Non-blocking comment, but can we convert these variables to camel case?

Reply:
Converted variables to camel case.